CorsixTH/Lua/dialogs/fullscreen/town_map.lua

```diff
@@ -186,15 +186,15 @@ function UITownMap:draw(canvas, x, y)
   self.info_font:draw(canvas, radiators,    x +  95, y + 265)
 
   -- Heating costs
-  local heating_costs = math.floor(((hospital.radiator_heat *10)* radiators)* 7.5)
+  local heating_costs = math.floor(((hospital.heatingController.radiator_heat *10)* radiators)* 7.5)
```

Comment:
Space on both sides of * (formatting was wrong before)

Reply:
I created a function for calculating the expected heating costs. I need the same calculation in the onEndDay and town_map